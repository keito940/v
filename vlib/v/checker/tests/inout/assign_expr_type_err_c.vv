fn main() {
	mut a := 'hello'
	a -= 'world'
}