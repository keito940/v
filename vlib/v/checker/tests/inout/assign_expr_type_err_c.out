vlib/v/checker/tests/inout/assign_expr_type_err_c.v:3:7: error: operator -= not defined on left type `string` and right type `string`
    1| fn main() {
    2|     mut a := 'hello'
    3|     a -= 'world'
                ~~~~~~~
    4| }
